packages/feature-toggle/src/constants.ts

```diff
@@ -0,0 +1,23 @@
+import type { ExtensionConfigKeys } from './types';
+
+export const extensionConfigKeys: ExtensionConfigKeys = {
+    AnnotationModeler: 'sap.ux.annotationModeler',
+    AppGenerator: 'sap.ux.appGenerator',
+    ApplicationModeler: 'sap.ux.applicationModeler',
+    Help: 'sap.ux.help',
+    RequirementsGathering: 'sap.ux.requirementsGathering',
+    ServiceModeler: 'sap.ux.serviceModeler',
+    Internal: 'sap.ux.internal'
+} as ExtensionConfigKeys;
+
+export const tokenToggleGuid: ExtensionConfigKeys = {
+    'dummy.test.testBetaFeatures.someTokenFeature': '77e0469d-1448-42bf-8d47-0944896dd9ed',
```

Comment:
What is `dummy.test.testBetaFeatures.someTokenFeature` used for?

Reply:
I think initially used as an example and then for testing. I can remove it